### PR TITLE
feat(#50): add GitHub action that synchronize changes from php-service-template repository

### PR DESCRIPTION
**Pull request by Derane**
## Description
add GitHub action that synchronize changes from php-service-template repository

## Related Issue

https://github.com/VilnaCRM-Org/php-service-template/issues/50

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING.md**](https://github.com/VilnaCRM-Org/php-service-template/blob/main/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] You have only one commit (if not, squash them into one commit).

## Summary by CodeRabbit

- **New Features**
	- Introduced a new automated workflow for synchronizing templates weekly on Mondays at 9 AM UTC or on manual trigger.
	- Automatically creates pull requests with a designated label for template updates.
	- Added documentation detailing the repository synchronization feature, including how it works, configuration instructions, and benefits of synchronization.
	- Implemented a `.templatesyncignore` file to exclude specific files from synchronization.

### CODE REVIEW

**Review by Kravalg — CHANGES_REQUESTED** [review]
It will be better to create PRs every week on Monday morning

**Comment by Kravalg** [comment]
P.S. Also we need to update readme.md and write that we have this possibility and how to configure it
The same for the php service template

File: README.md [hunk 1]
```diff
@@ -121,6 +121,24 @@
...
+
+By default, the workflow runs every Monday at 9:00 AM UTC. You can also manually trigger it from the GitHub Actions tab.
+
+### Configuration
+
+1. The synchronization is managed through a GitHub Actions workflow, which is triggered automatically via cron or manually.
+2. The `source_repo_path` must point to the repository you want to sync from (e.g., `VilnaCRM-Org/php-service-template`).
+3. Make sure you have the necessary permissions set up for the GitHub token to allow synchronization.
+
+You can see a sample configuration for the synchronization workflow, stored in `.github/workflows/template-sync.yml`.
```
[the remark this answers is not in the document]
Comment (Derane) [reply README.md:140]: Could you help me? Found the link and enhanced  it.

File: README.md [hunk 2]
```diff
@@ -121,6 +121,29 @@ If the documentation doesn't cover what you need, search the [many questions on
...
+## Repository Synchronization with Template
+
+We have integrated an automated repository synchronization feature using the [actions-template-sync](https://github.com/marketplace/actions/actions-template-sync) GitHub Action. This allows the repository to stay in sync with a designated template repository.
+
+### How It Works
+
+This workflow automatically creates a pull request in this repository whenever changes are detected in the template repository, ensuring that the latest updates from the template are applied.
+
+By default, the workflow runs every Monday at 9:00 AM UTC. You can also manually trigger it from the GitHub Actions tab.
+
```
[the remark this answers is not in the document]
Comment (Kravalg) [reply README.md:133]: Nice idea, @Derane please write the link there to the GitHub actions tab

**Review by Kravalg — CHANGES_REQUESTED** [review]
Good job
Please add this files to the php service template and update the wiki as well

**Review by Kravalg — CHANGES_REQUESTED** [review]
Why do we need to ignore so much files ?
I agree with composer.json, but why do we need other files there?

**Review by Kravalg — APPROVED** [review]
Looks good 👍

**Pull request merged**